src/anomalib/data/base/dataset.py

```diff
@@ -84,15 +84,21 @@ def __len__(self) -> int:
         """Get length of the dataset."""
         return len(self.samples)
 
-    def subsample(self, indices: Sequence[int], inplace: bool = False) -> "AnomalibDataset":
+    def subsample(
+        self,
+        indices: Sequence[int],
+        inplace: bool = False,
+    ) -> "AnomalibDataset":
```

Comment:
This change is not needed
```suggestion
    def subsample(self,  indices: Sequence[int],  inplace: bool = False) -> "AnomalibDataset":
```